common/remote/sockfile.cpp

```diff
@@ -1052,7 +1086,7 @@ class CRemoteBase: public CInterface
                         WARNLOG("Remote file authenticate %s for %s ",e->errorMessage(err).str(),ep.getUrlStr(eps.clear()).str());
                         e->Release();
                         if (!retries)
-                            break;
+                            break; // MCK - is this a warning or an error ? If an error, should we close and throw here ?
```

Comment:
Also (separate JIRA probably), it would be more logical I'd have thought if connectSocket returned socket.
As it stands, afaics, sendRemoteCommand calls connectSocket, then has to loop around to find the socket in ConnectionTable, connectSocket just added it to. Then it checks if socket still alive (i.e. the socket it's just created), then loops around to actually send the data.
I would expect sendRemoteCommand calling code to look something like:
```
    socket.setown(connectSocket(tep));
```